hoodie-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/realtime/HoodieRealtimeRecordReader.java

```diff
@@ -125,7 +125,7 @@ private void readAndCompactLog() throws IOException {
     HoodieCompactedLogRecordScanner compactedLogRecordScanner =
         new HoodieCompactedLogRecordScanner(FSUtils.getFs(), split.getBasePath(),
             split.getDeltaFilePaths(),
-            readerSchema, split.getMaxCommitTime());
+            readerSchema, split.getMaxCommitTime(), (long) 0.75*jobConf.getMemoryForMapTask());
```

Comment:
@vinothchandar here, will make the change to make 0.75 a constant ? Or can we just pick it up from jobConf and default to some value if not present ?

Reply:
yes a job conf would be good..

Reply:
Yes, added a CONF but we probably need to document and expose this conf properly since there should be an easy way to set this for Presto/Hive queries.

Reply:
sg